Tests/CommonHelpers/Acts/Tests/CommonHelpers/CylindricalTrackingGeometry.hpp

```diff
@@ -46,10 +47,126 @@ struct CylindricalTrackingGeometry {
       std::reference_wrapper<const GeometryContext> gctx)
       : geoContext(gctx) {}
 
+  using DetectorStore = std::vector<std::unique_ptr<const DetectorElementStub>>;
+
   /// The detector store for memory management
-  std::vector<std::unique_ptr<const DetectorElementStub>> detectorStore = {};
+  DetectorStore detectorStore = {};
+
+  /// Generartor of surfaces for a ring
```

Comment:
Could you add some explanations for the arguments?

Reply:
Fixed.